### ISSUE TITLE
The npm package lacks the types definition

### ISSUE THREAD

**Issue by ericmorand** [Hypnos3/suncalc3#4]
Although the types are present in the project and are declared in `package.json`, they are missing from the published npm package, at least in 2.0.5:

https://www.npmjs.com/package/suncalc3?activeTab=explore

**Comment by SamLeatherdale** [comment]
Yep, to fix this they need to include `suncalc.d.ts` in the `files` array: https://github.com/Hypnos3/suncalc3/blob/master/package.json#L47
